core/src/test/scala/com/qvantel/jsonapi/PaginationSpec.scala

```diff
@@ -0,0 +1,85 @@
+package com.qvantel.jsonapi
+
+import com.netaporter.uri.dsl._
+import org.specs2.mutable._
+import spray.json.DefaultJsonProtocol._
+import spray.json._
+import com.netaporter.uri.Uri
+import spray.json
+
+class PaginationSpec extends Specification {
+  implicit val apiRoot: com.qvantel.jsonapi.ApiRoot = ApiRoot(Some("/api"))
+
+  @jsonApiResource final case class Test(id: String, attr: String)
+
+  "pagination links" >> {
+    "works for simplest case" >> {
+      implicit val pagination: JsonApiPagination =
+        JsonApiPagination(Uri.parse("/")).withNext("number" -> "2", "size" -> "5")
+
+      val entities = Seq(Test("id", "attr"))
+
+      val json = rawCollection(entities)
+      println(json.getFields("links").head.prettyPrint)
```

Comment:
A lot of println() here but I guess you kept them for logging purposes?

Reply:
No, they shouldn't be there. Removed now.